test/cmdlineTests/wasm_to_wasm/output

```diff
@@ -0,0 +1,27 @@
+
+======= wasm_to_wasm/input.yul (Ewasm) =======
+
+Pretty printed source:
+object "object" {
+    code {
+        function main()
+        { i64.store8(0x01:i32, 42) }
+    }
+}
+
+
+Binary representation:
+0061736d01000000010401600000020100030201000503010001060100071102066d656d6f72790200046d61696e00000a11010f01007e02404201a7422a3c00000b0b
```

Comment:
This test would be a lot more meaningful if it included the decoded binary. I.e. the output `wasm2wat` (from wabt) produces. Running `wasm-validate` as a part of our tests would also be helpful.

Reply:
Once we have the semantic tests running, those do wasm validation as well.

Reply:
Great. That will make working with it much easier.